zebra-chain/src/orchard/keys.rs

```diff
@@ -278,6 +280,8 @@ impl Eq for SpendValidatingKey {}
 
 impl From<[u8; 32]> for SpendValidatingKey {
     fn from(bytes: [u8; 32]) -> Self {
+        // Review question: @dconnolly do we need to delete or fix this panic?
+        // It's used as part of the full viewing key, which is used to generate new spending keys.
```

Comment:
@dconnolly can you check if this code is a security risk, because it panics?

Reply:
This should be an `impl TryFrom` because the conversion from an arbitrary array of 32 bytes may not be a valid key; so instead of panic'ing it should return a `Result::Err`

Reply:
Thanks, I moved it to a test-only module.